orchagent/muxorch.cpp

```diff
@@ -433,6 +433,12 @@ bool MuxCable::stateStandby()
 
 void MuxCable::setState(string new_state)
 {
+    if (strcmp(new_state.c_str(), muxStateValToString.at(state_).c_str()))
```

Comment:
For this, we still need to update the HW MuX cable state at line 455. So can you move this to 456 and else part have the error log